src/components/CompareResults/RevisionRow.tsx

```diff
@@ -90,6 +90,7 @@ const stylesLight = {
       },
       '.total-runs': {
         backgroundColor: Colors.Background200,
+        gap: '8px',
```

Comment:
you also need to add it to the dark section below

This repetition is a bit of a bummer, we should be able to reduce the duplication with eg a function taking the color as a parameter and returning the values. (not for this PR)

Reply:
Ah good catch, thanks! Updated it.